proto/mysqlctl.proto

```diff
@@ -64,6 +70,7 @@ service MysqlCtl {
   rpc ApplyBinlogFile(ApplyBinlogFileRequest) returns (ApplyBinlogFileResponse) {};
   rpc ReinitConfig(ReinitConfigRequest) returns (ReinitConfigResponse) {};
   rpc RefreshConfig(RefreshConfigRequest) returns (RefreshConfigResponse) {};
+  rpc VersionString(VersionStringRequest) returns (VersionStringResponse) {};
```

Comment:
This adds to `mysqlctl`, but that is always upgraded together with `vttablet` which is what uses this API, so this should be all safe to use as well. The only usage is from either `mysqlctl` itself or `vttablet`.